lib/src/iso/bmff/boxes/AudioRenderingIndicationBox.ts

```diff
@@ -0,0 +1,13 @@
+import type { FullBox } from './FullBox.js';
+
+/**
+ * ISO/IEC 14496-12:202x - 12.2.8 Audio rendering indication box
+ *
+ * @group ISOBMFF
+ *
+ * @beta
+ */
+export type AudioRenderingIndicationBox = FullBox & {
```

Comment:
(optional) potentially useful to prettify the intersection by using `Prettify`, where `Prettify` is:

```ts
export type Prettify<Type> = Type extends Function
  ? Type
  : Extract<
      {
        [Key in keyof Type]: Type[Key];
      },
      Type
    >;
```

See examples in https://github.com/ts-essentials/ts-essentials/blob/master/lib/prettify/README.md